src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -161,8 +159,10 @@ public static int parseClaimAmount(String claimAmount) throws ParseException {
         requireNonNull(claimAmount);
         String[] claimAmountString = claimAmount.trim().split("\\.");
 
-        if (claimAmountString.length != 2) {
-            throw new ParseException(MESSAGE_TOO_MANY_DECIMALS);
+        if (claimAmountString.length > 2) {
+            throw new ParseException(Claim.MESSAGE_TOO_MANY_DECIMALS);
+        } else if (claimAmountString.length == 1) {
```

Comment:
Enforces the use of cents i.e. `500` will not be allowed, but `500.0` and `500.00` will be allowed.

Reply:
i think this was fixed further later